### DISCUSSION TITLE
Cannot read error messages (as they do not fit in the command line).

### DISCUSSION THREAD

**Discussion by 7ombie** [helix-editor/helix#7421] [Troubleshooting] [answered]
I'm sure I'm missing something, but I cannot read error messages in Helix. For example, if I run `:config-reload` when there's an error in the config file, Helix writes an error message to the command line, concatenating multiple lines together and truncating anything after the 120th character (the width of my terminal).

<img width="1280" alt="Screenshot 2023-06-22 at 00 23 37" src="https://github.com/helix-editor/helix/assets/80077603/351932d4-f36f-4fd6-a8b8-cb49ee83e680">

I can't find anything online about this, so assume I'm missing something. Is there a way to expand and read these messages (inside Helix)?

**Comment by gabydd** [comment, marked as answer]
The message will be also added to the log file which you can open with ``:log-open``

**Reply by 7ombie** [reply]
Thanks. This is the correct answer. I was just hopeful there was something a bit more user-friendly.

The log file doesn't update while it's open, even if you run`:log-open` again. It's also editable, which isn't ideal.

It'd be nice if we could open a read-only buffer that was kept in sync with the log over time. Maybe View Mode could be adapted to allow any file to be viewed readonly while tracking changes on disk?? It'd also be useful if it had an option for keeping the cursor on the last line.
